megamek/src/megamek/common/Crew.java

```diff
@@ -294,10 +369,28 @@ public String getNickname(int pos) {
         return nickname[pos];
     }
 
+    public int getGender() {
+        return gender[0];
+    }
+
+    public int getGender(int pos) {
+        // The randomize return value is used in MekHQ to create new personnel following a battle,
+        // and should not be changed without ensuring it doesn't break on that side
+        if (pos < gender.length) {
+            return gender[pos];
+        } else {
+            return G_RANDOMIZE;
+        }
+    }
+
+    public static int getGenderAsInt(boolean isFemale) {
```

Comment:
What's the deal with this function? Why not just either pass in ints or switch the whole thing to a boolean, why convert between the two?

Reply:
Temporary function to keep functionality while I finished updating names. I'll be removing this in wave 2.